docs/core/deploying/native-aot/index.md

```diff
@@ -17,7 +17,7 @@ The Native AOT deployment model uses an ahead-of-time compiler to compile IL to
 
 # [Windows](#tab/windows)
 
-[Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including the **Desktop development with C++** workload with all default components.
+The [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/), or you can install [[Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including the **Desktop development with C++** workload. Open a **Native Tools Command Prompt** to ensure the required tools are on the PATH.
```

Comment:
> Open a **Native Tools Command Prompt** to ensure the required tools are on the PATH.

This should not be required. It is not required in the sample dockerfile: https://github.com/dotnet/samples/blob/main/core/nativeaot/HelloWorld/Dockerfile.windowsservercore-x64

The golden path has to be plain `dotnet publish` with no special requirements on the environment.

We used to require "Native Tools Command Prompt". It produced a constant stream of issues (several issues per month). We fixed that by auto-detecting the VS installed on the machine. We do not want to go back and reintroduce the special environment requirements.

Reply:
Examples of issues that were caused by "Native Tools Command Prompt" requirement:

https://github.com/dotnet/corert/issues/6282
https://github.com/dotnet/corert/issues/7871
https://github.com/dotnet/corert/issues/6885
https://github.com/dotnet/corert/issues/588